root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Sogo.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$L['Sogo_header'] = 'Configuration';
+$L['Sogo_Title'] = 'SOGo';
+$L['Sogo_Description'] = 'SOGo is the groupware solution';
+$L['status_label'] = 'Enable SOGo';
+$L['Dav_label'] = 'Enable CalDAV and CardDAV';
+$L['MailAuxiliaryUserAccountsEnabled_label'] = 'Allow Users to add other IMAP accounts';
+$L['ActiveSync_label'] = 'Enable Microsoft ActiveSync';
+$L['AdminUsers_label'] = 'Administrators';
+$L['Advanced_label'] = 'Advanced settings';
+$L['VirtualHost_label'] = 'Make SOGo reachable only from this domain name list';
+$L['Notifications_label'] = 'Notifications';
+$L['ACLs_label'] = 'Sent by emails to involved users of a calendar/address book';
+$L['Folders_label'] = 'Sent by emails on the creation of a calendar/address book.';
+$L['Appointment_label'] = 'Sent by emails to meeting participants';
```

Comment:
I think 
`Send by emails to involved users of a calendar/address book`
can be better phrased:
`Send email to involved users of a calendar/address book `
`Send email on the creation of a calendar/address book`
`Send email to meeting participants`